getting_started/build-guide/build-from-scratch.rst

```diff
@@ -365,9 +378,21 @@ can skip the next two sections.
 The installer also builds and deploys some user-space binaries from the
 ``sof/tools/`` subdirectory.
 
+.. note::
+
+   If interrupted, the installer process might leave corrupted files
+   in the ``installer-builds`` subdirectory. In this case, all
+   subsequent attempts to run the installer end with an error. To
+   resolve this issue, delete the ``installer-builds`` directory and
+   run the installer again.
```

Comment:
By chance I got for the first time ever in the same error message you reported to me by email. It didn't happen because the build was interrupted but because I tried to switch toolchains without deleting the build directory. This should be very rare and all developers' first reflex when they don't understand some build error is to delete the build directorie(s).

Did you ever experience that same error again?

In any case the installer is merely invoking the other build scripts, so if you really want to add such a warning it should not sound like the installer builds in a unique and different way. How about something like this:

> The installer is much faster than the lower level `./scripts/` it relies on because it does not start by deleting the build directories every time it is invoked.  However some "big" configuration changes like switching to a different toolchain or some rare build failures can leave the `installer-builds/build_*` directories in an inappropriate state. In such a case just delete them and run the installer again.